playbooks/group_vars/webportals.yml

```diff
@@ -125,6 +125,7 @@ webportal_docker_compose_files_optional:
 webportal_docker_compose_files_dict:
   a: "docker-compose.accounts.yml"
   j: "docker-compose.jaeger.yml"
+  m: "docker-compose.mongodb.yml"
```

Comment:
We need to add `docker-compose.mongodb.yml` also to `webportal_docker_compose_files_optional` list above.
(Didn't find a way to do it via suggestion.)